.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+Please include a summary of the change and (optionally) which issue is fixed. Please also include
+relevant motivation and context.
+
+Fixes # (issue)
+
+## Type of change
+
+Please check the options below:
+
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
```

Comment:
should i be checking "new feature" and "breaking change" if i add a new feature that breaks things? if so, remove "non-breaking change" here and line above. if not, then split "breaking change" into breaking bugfix/breaking feature